### ISSUE TITLE
A bunch of feature requests.

### ISSUE THREAD

**Issue by coxtor** [GnomeSnapExtensions/gSnap#42]
Thanks so much - i love it. its simple and straight forward and usable without the keyboard - which is important to me! Using it with my new  49" 5k ultra wide screen.

Few feature suggestions: 
- make tabbe items draggable into new spaces
- make it possible to delete layouts from the ui
- give an option to have a "minilayout popup" under the mouse and be able to drag the window into the section in the minipopup rather than drawing the layout onto the whole screen (similar to dell display manager) 
- 
- option for automatic layoutswitching dynamic layouts based on connected screens
-  dynamic layouts based on the number of open windows

**Comment by meronz** [comment]
These are all good suggestions, some easier to implement than others. I'll leave this issue open, but if anyone wants to tackle one issue, feel free to send in PRs!

**Comment by tommiedepommie** [comment]
Hello, I'm a new user of Gsnap and using it on a 43" screen. I made my own layout, 3columns x 2 rows. The middle column is twice the width of the columns on the side. It would be nice if it was possible to stretch a used tile/window over an empty one so the window would double in size. (using two tiles) As a new user I'm still experimenting with this great tool. Thank You. Happy new year !!!!

**Comment by tomByrer** [comment]
> if it was possible to stretch a used tile/window over an empty one so the window would double in size. (using two tiles

Yes, MicroSoft PowerToys does this; while dragging, you hover mouse cursor over the border of 2 zones/itles, & it will fill both zones.  I find this very handy, since it effectively triples the zones: make several smaller zones, yet bigger zones are available anytime.

**Comment by williamdean19** [comment]
@tommiedepommie this is now possible with the option in the settings, enable the setting for "Hold ALT to span multiple zones". However, it doesn't work if the "Show tabs" option is enabled, so you have to pick which feature you want more. 

@meronz I would really like a keyboard-only option to enable zone spanning, since the feature I mentioned does not seem to work with the super+left/right/up/down shortcuts. For example if I hold down alt then press super+left, I would like the focused window to expand to the left. Currently, this does not work and can only be done with the mouse. (default OS behavior for alt+super+left is to switch workspaces, so it might need a different key combo to avoid conflicts) 

Appreciate your work on this app, it was definitely one of the main things  I needed to switch off of Windows, I have gotten so used to FancyZones! Thank you.
